programmers/위장.js

```diff
@@ -0,0 +1,15 @@
+function solution(clothes) {
+    const obj = clothes.reduce(function(cloth, kind) {
+        cloth[kind[1]] = cloth[kind[1]] ? cloth[kind[1]] + 1 : 1;
+        return cloth;
+    }, {});
+    
+    const value = Object.values(obj);
+    
+    let answer = 1;
+    
+    if (value.length === 1) return value[0];
+    else value.forEach(el => answer *= (el + 1));
+
+    return answer - 1;
```

Comment:
```suggestion
    if (value.length === 1) answer = value[0] + 1;
    else value.forEach(el => answer *= (el + 1));

    return answer - 1;
```
이렇게 해야되는건지 잘은 모르겠지만
뭔가 solution이라는 function 안에 return이 2개인 것보다는
하나로 return 값을 통일해 주는것이 더 좋을거같다는 생각이 드네요